### ISSUE TITLE
MORE_DROPS does not work 

### ISSUE THREAD

**Issue by hkkongou** [GC-spigot/AdvancedEnchantments#2272]
## Details

**Describe the bug**
similar to #2159 
kill mobs with more_drops does nothing, but this time in 1.16.5

**To Reproduce** 
use 
```
diploid:
  display: '%group-color%Diploid'
  description: "Chance to multiply mob drops.\nCosts 5 souls / activation."
  applies-to: 'Swords'
  type: 'KILL_MOB'
  group: 'FABLED'
  applies:
    - ALL_SWORD
  levels:
    1:
      effects:
        - "MORE_DROPS:5"
        - "MESSAGE:activated"
```

on swords, hit or kill mobs.

**Screenshots / Video**

## Server Information
- "/ae plinfo" link:  https://paste.md-5.net/iqukayacal
no error shown in console

tested with AE only too
![image](https://user-images.githubusercontent.com/61608872/129430937-83ce1c5b-7bbe-49c0-8b35-5c8e591ced55.png)

**Comment by Insprill** [comment]
Couldn't reproduce but I made some changes to it so please let me know if 8.6.6 fixes it when it releases

**Comment by SenPr** [comment]
Similar problem here. It works, but it does not give the original drop.

**Comment by SenPr** [comment]
Seems to be fixed in the update 8.6.6
